libs/checkpoint-sqlite/langgraph/checkpoint/sqlite/__init__.py

```diff
@@ -56,7 +56,7 @@ class SqliteSaver(BaseCheckpointSaver[str]):
         >>> builder.add_node("add_one", lambda x: x + 1)
         >>> builder.set_entry_point("add_one")
         >>> builder.set_finish_point("add_one")
-        >>> conn = sqlite3.connect("checkpoints.sqlite")
+        >>> conn = sqlite3.connect("checkpoints.sqlite", check_same_thread=False)
```

Comment:
Would you mind adding an inline explanation explaining why this is OK to do in the context of langggraph. This assumes that writes are sequential and are not in parallel